### ISSUE TITLE
API PageSize has no effect except List Conversation API

### ISSUE THREAD

**Issue by satsangswami** [freescout-helpdesk/freescout#3433]
* List Api of conversation sends documents as per the pageSize Parameter. However, other list apis always sends 50 documents, regardless of the pageSize Parameter.

PHP version: 8.0.30
FreeScout version: 1.8.80
Api & Webhook Module version: 1.0.58

**Comment by satsangswami** [comment]
Solution:
Change these lines
```

$customers = $query->paginate(self::PAGE_SIZE);
$customers = $query->paginate($request->pageSize ?: self::PAGE_SIZE);

$users = $query->paginate(self::PAGE_SIZE); 
$users = $query->paginate($request->pageSize ?: self::PAGE_SIZE);

$custom_fields = $query->paginate(self::PAGE_SIZE); 
$custom_fields = $query->paginate($request->pageSize ?: self::PAGE_SIZE);

$mailboxes = $query->paginate(self::PAGE_SIZE);
$mailboxes = $query->paginate($request->pageSize ?: self::PAGE_SIZE);

$tags = $query->paginate(self::PAGE_SIZE);
$tags = $query->paginate($request->pageSize ?: self::PAGE_SIZE);

$timelogs = $query->paginate(self::PAGE_SIZE);
$timelogs = $query->paginate($request->pageSize ?: self::PAGE_SIZE);
```

**Comment by freescout-helpdesk** [comment]
Fixed in API & Webhooks Module v1.0.71
